### ISSUE TITLE
Create a simple and achievable PR template 

### ISSUE THREAD

**Issue by authorjapps** [authorjapps/zerocode#337]
AC1:
The template can reside in the `doc` folder of the project root directory. This makes it visible to all.
Help on [how to create a PR template](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)

AC2:
Please have the rules which can be reasonably achieved by a developer.
The following(at least) should be the part of the template.
```
PR Branch
**_#ADD LINK TO THE PR BRANCH_**

Unit tests added?: (Y/N)

Integration tests added?: (Y/N)

Test names are meaningful?: (Y/N)

Feature manually tested?: (Y/N)

Branch build passed?: (Y/N)

No 'package.*' in the imports? (Y/N)

Relevant Wiki page updated with clear instruction for the end user? (Y/N)

Http test added to `http-testing` module(if applicable) ?: (Y/N)

Kafka test added to `kafka-testing` module(if applicable) ?: (Y/N)

```

Related article in [Medium](https://medium.com/better-programming/github-templates-the-smarter-way-to-formalize-pull-requests-among-development-teams-89f8d6a204f) on PR Template

**Comment by hgvanpariya** [comment]
@authorjapps : Here I've added PR template, please provide your feedback for the same.

https://github.com/authorjapps/zerocode/pull/338

**Comment by authorjapps** [comment]
Available in version `1.3.17`
